pkg/apply/applier.go

```diff
@@ -42,11 +44,6 @@ func NewApplier(factory util.Factory, ioStreams genericclioptions.IOStreams) *Ap
 	}
 }
 
-// poller defines the interface the applier needs to poll for status of resources.
-type poller interface {
-	Poll(ctx context.Context, identifiers []object.ObjMetadata, options polling.Options) <-chan pollevent.Event
-}
-
 // Applier performs the step of applying a set of resources into a cluster,
```

Comment:
We should probably put a couple paragraphs of comments in the applier describing how the different pieces work together in the "queue of tasks".